aptos-move/framework/aptos-token-objects/sources/token.move

```diff
@@ -51,6 +55,13 @@ module aptos_token_objects::token {
         mutation_events: event::EventHandle<MutationEvent>,
     }
 
+    #[resource_group_member(group = aptos_framework::object::ObjectGroup)]
+    /// Represents first addition to the common fields for all tokens
+    struct TokenAppendix1 has key {
+        /// Unique identifier within the collection, optional, 0 means unassigned
+        index: AggregatorSnapshot<u64>,
+    }
```

Comment:
I'd prefer we just call this struct `Index` or `TokenIndex`. `TokenAppendix1` has no useful information about it and we're not going to be able to add more fields anyway.